test/headless/specs/utilities.js

```diff
@@ -19,8 +21,9 @@ testModule('elementSatisfiesRatio', class extends TestClass {
         window.STATE.satisfied = true;
       })
     })
-    .wait(30)
+    .wait(constants.time.RAF_THRESHOLD * 5)
```

Comment:
When we give small time interval test cases are breaking even though the case which we are testing is working good. When we give a bit more time by using the wait. `assert()` is not able to get the right value. So I have added a padding time every place we do a scroll so that assert() get correct value. 
We need to note on important thing here, sometime all test cases work perfectly even though we don't put wait. So it not consistently reproducible. But by giving padding time solves problem.